cluster/update-storage-objects.sh

```diff
@@ -59,7 +61,12 @@ for resource in "${resources[@]}"
 do
   for namespace in "${namespaces[@]}"
   do
+    # If get fails, assume it's because the resource hasn't been installed in the apiserver.
+    # TODO hopefully we can remove this once we use dynamic discovery of gettable/updateable
+    # resources.
+    set +e
```

Comment:
It looks like depending on what you have in `KUBE_API_VERSIONS` and `RUNTIME_CONFIG`, you either get `Unable to list "jobs": the server could not find the requested resource` (if extensions/v1beta1 is disabled but batch/v1 is enabled) or `the server doesn't have a resource type "hpas"` (if you try to request an invalid resource name). Maybe we allow the latter but fail if any other error shows up?